src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+
+  if (days >= 3 && days <= 6) {
+    return price - 20;
+  }
+
+  if (days >= 7) {
+    return price - 50;
+  }
```

Comment:
Use variables for the main values so that you can reuse them and give them descriptive names